style.css

```diff
@@ -41,7 +48,68 @@ footer .fab:hover {
   font-size: 18px;
 }
 
-@media screen and (min-width: 762px) {
+.bg-light1 {
+  color: #f72210;
+  background-color: rgba(255, 255, 255, 0.9);
+}
+
+.bg-light2 {
+  color: #3581f4;
+  background-color: rgba(255, 255, 255, 0.9);
+}
+
+.bg-light3 {
+  color: black;
+  background-color: #c2d4d7;
+}
+
+.width {
+  width: 120px;
+  height: 140px;
+}
+
+.vs {
+  position: absolute;
+  left: 45%;
+  right: 45%;
+  bottom: -16px;
+  z-index: 1;
+  width: 32px;
+  height: 32px;
+  background-color: #000;
+  color: #d6d6d6;
+  border-radius: 50%;
+  text-align: center;
+  font-weight: bold;
+}
```

Comment:
- [ ] [OPTIONAL] avoid using rules that you can already apply using bootstrap.
- In this case `position: absolute;`, ` text-align: center;`, `font-weight: bold;` can be used with bootstrap